src/main/java/org/moditect/jfrunit/JfrEvents.java

```diff
@@ -305,4 +326,19 @@ private String getDumpFileName() {
     private String getDefaultDumpFileName() {
         return testMethod.getDeclaringClass().getName() + "-" + testMethod.getName() + ".jfr";
     }
+
+    public JfrAnalysisResults automaticAnalysis() {
```

Comment:
Perhaps `awaitEvents()` call could be part of the method as this is likely what user would always do manually anyway.

Reply:
I have moved `awaitEvents()` into `automaticAnalysis()`